src/main/resources/citizen/templates/document/claimIssueReceipt.html

```diff
@@ -0,0 +1,372 @@
+          {{ claimant.fullName }}
+        </td>
+      </tr>
+      {% if claimant.contactPerson is not empty %}
+      <tr>
+        <td>Contact person:</td>
+        <td>
+          {{ claimant.contactPerson }}
+        </td>
+      </tr>
+      {% endif %}
+
+      {% if claimant.businessName is not empty %}
+      <tr>
+        <td>Business name:</td>
+        <td>
+          Trading as {{ claimant.businessName }}
+        </td>
+      </tr>
+      {% endif %}
+      <tr>
+        <td>Address:</td>
+        <td>
+          {{ claimant.address.line1 }}
+          <br/>
+          {% if claimant.address.line2 is not empty %}
+          {{ claimant.address.line2 }}
+          <br/>
+          {% endif %}
+          {{ claimant.address.city }}
+          <br/>
+          {{ claimant.address.postcode }}
+        </td>
+      </tr>
+      {% if claimant.correspondenceAddress is not empty %}
+      <tr>
+        <td>Correspondence address:</td>
+        <td>
+          {{ claimant.correspondenceAddress.line1}}
+          <br/>
+          {% if claimant.correspondenceAddress.line2 is not empty %}
+          {{ claimant.correspondenceAddress.line2 }}
+          <br/>
+          {% endif %}
+          {{ claimant.correspondenceAddress.city }}
+          <br/>
+          {{ claimant.correspondenceAddress.postcode }}
+        </td>
+      </tr>
+      {% endif %}
+      <tr>
+        <td>Email:</td>
+        <td>
+          {{ claimant.email }}
+        </td>
+      </tr>
+    </table>
+  </div>
+</div>
+
+<div class="section">
+  <div class="section-heading">
+    <strong>Defendant’s details</strong>
+  </div>
+  <div class="section-body">
+    <table>
+      <tr>
+        <td>Name:</td>
+        <td>
+          {{ defendant.fullName }}
+        </td>
+      </tr>
+      {% if defendant.businessName is not empty %}
+      <tr>
+        <td>Business name:</td>
+        <td>
+          Trading as {{ defendant.businessName }}
+        </td>
+      </tr>
+      {% endif %}
+      <tr>
+        <td>Address:</td>
+        <td>
+          {{ defendant.address.line1 }}
+          <br/>
+          {% if defendant.address.line2 is not empty %}
+          {{ defendant.address.line2 }}
+          <br/>
+          {% endif %}
+          {{ defendant.address.city }}
+          <br/>
+          {{ defendant.address.postcode }}
+        </td>
+      </tr>
+      {% if defendant.email is not empty %}
+      <tr>
+        <td>Email:</td>
+        <td>
+          {{ defendant.email }}
+        </td>
+      </tr>
+      {% endif %}
+    </table>
+  </div>
+</div>
+
+<div class="section">
+  <div class="section-heading">
+    <strong>Claim amount</strong>
+  </div>
+  <div class="section-body">
+    <table>
+      <tr>
+        <td>Claim amount:</td>
+        <td>
+          {{ claim.claimAmount }}
+        </td>
+      </tr>
+      {% if claim.interest is defined %}
+      <tr>
+        <td>Interest rate:</td>
+        <td>
+          {{ claim.interest.rate }}a year
```

Comment:
shouldn't there be a space after `}}`?

Reply:
There should be, great eye. Changed now and will be pushed.

Reply:
Done